### PR TITLE
feat(statics): add AVAXC coin to statics

### PR DESCRIPTION
**Pull request by CapnMigraine**
Add support for AVAX on the C-Chain as AVAXC

CLOSES TICKET: STLX-5084

### CODE REVIEW

**Review by smaria95 — DISMISSED** [review]
LGTM

**Review by jrafael-simtlix — APPROVED** [review]

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -53,6 +53,8 @@ export const coins = CoinMap.fromCoins([
     ALGO_FEATURES,
     KeyCurve.Ed25519
   ),
+  account('avaxc', 'Avalanche C-Chain', Networks.main.avalancheC, 18, UnderlyingAsset.AVAXC, ETH_FEATURES),
+  account('tavaxc', 'Testnet Avalanche C-Chain', Networks.main.avalancheC, 18, UnderlyingAsset.AVAXC, ETH_FEATURES),
```
Comment (tylerlevine) [inline modules/statics/src/coins.ts:57]: Shouldn't the network be `avalanchCTestnet` here?

**Review by tylerlevine — CHANGES_REQUESTED** [review]
Please check tests

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (CapnMigraine) [reply modules/statics/src/coins.ts]: it should be `Networks.test.avalancheC`
Fixed

**Review by tylerlevine — APPROVED** [review]

**Review by jrafael-simtlix — APPROVED** [review]

**Review by smaria95 — APPROVED** [review]
LGTM

**Pull request merged**